### DISCUSSION TITLE
milvus server 版本1.0.0，pymilvus可以使用1.1.0吗

### DISCUSSION THREAD

**Discussion by wlj521** [milvus-io/milvus#7132] [Q&A and General discussion] [answered]
milvus server 版本1.0.0，pymilvus可以使用1.1.0吗

**Comment by wlj521** [comment]
Version of python SDK(1.1.0) not match that of server1.0.0

**Comment by yhmo** [comment, marked as answer]
Basically, sdk 1.1.x must compatible with milvus 1.1.x, but we don't promise sdk 1.1.x can compatible with milvus 1.0.x.

**Reply by wlj521** [reply]
我试了，这个版本不能向下兼容

**Reply by yhmo** [reply]
我们强制对sdk版本和server版本做了检查，sdk 1.0.x只能匹配server 1.0.x， 不然会报错。实际上1.1版本和1.0版本仅仅是改动了2个api，增加了1个api，对其他api其实可以兼容的，只不过我们做了检查，是为了防止使用时出现一些奇怪的问题。

**Reply by wlj521** [reply]
我个人觉得可以兼容一下，可能出现的问题做一个提示，如果server端已经发布了1.0.0版本并且在使用中，要升级到1.1.1，客户端就不好处理
